variables.tf

```diff
@@ -46,6 +46,12 @@ variable "aliases" {
   default     = []
 }
 
+variable "use_regional_s3_endpoint" {
+  type        = "string"
+  description = "When set to 'true' the s3 origin_bucket will use the regional endpoint address instead of the global endpoint address"
+  default     = "false"
```

Comment:
Is this just to not introduce a change? Since the regional endpoint is an improvement shouldn't this be true always?

Reply:
Yeah. Backwards compatibility and non-breaking changes are good.

But the README can be updated to say that they should prefer it.